docs/kb/advanced-certificates.md

```diff
@@ -63,3 +63,98 @@ LS0tLS1CRUd......==
     * Custom VMware Event Broker Appliance TLS Certificate Authority Certificate (Base64)
 
 3. Power on the VMware Event Broker Appliance and ensure that the provided TLS certificate is now used instead of the auto-generated self-sign TLS certificate by opening a browser to one of the VMware Event Broker Appliance endpoints such as `/status`.
+
+## Adding a Trusted Root Certificate to VEBA
+
+For organizations that require the use of a Private or Enterprise Certificate Authority (CA) and wish to have full TLS trust when the VMware Event Broker Appliance is configured with a vCenter Server that has a TLS certificate generated by the internal CA, a root of trust must be established.
+
+To support this use case, a post-deployment reconfiguration of the VMware Event Broker Appliance is required to import the trusted root certificate of your internal certificate authority. 
+
+In order to import your trusted root certificate, please follow the steps outlined below.
+
+### Assumptions
+
+* Pre-download you root certificates from your internal certificate, which must be .PEM encoded
+
+In the example, the downloaded root certificate file is named `ca.crt`
+
+### Steps
+
+Step 1 - Transfer or copy the contents of the root certificate to the VMware Event Broker Appliance and create a new kubernetes configMap from that file.
+
+```console
+kubectl -n vmware-system create cm ca-cert --from-file ca.crt
```

Comment:
Below the file name is vc-ca.cert which should not have worked in your case then...

Reply:
This was a copy/paste error for docs. Although we used vCenter's root CA, customers would be using their actual root CA, so I didn't want to call this `vc-ca` but rather `ca-cert`